pipelines/tenant/update-internal-services-manager/README.md

```diff
@@ -0,0 +1,12 @@
+# update-internal-services-manager
+
+Tekton pipeline to execute the update the internal-services manager yaml to the latest image in the
```

Comment:
"to execute the update the internal-services manager yaml" - looks like a typo. Either "the update of" or just "to update".

Reply:
fixed

Reply:
Please also fix this in the yaml.

Reply:
oh yeah, nice catch. fixed